javascript-practice/src/javascripts/models/product.js

```diff
@@ -0,0 +1,21 @@
+import APIService from "../services/service"
+
+export default class ProductModel {
+	constructor(id, title, price, description, categoryId, dimension, image) {
```

Comment:
So in you need to init all of params when you create this instance. Should not do that

Reply:
You missed that, @mthanh2209 ?